source/renderer/app/api/api.js

```diff
@@ -211,7 +212,70 @@ export default class AdaApi {
     this.config = config;
   }
 
-  getWallets = async (request: {
+  /* eslint-disable */
+  getWallets = async (request: { isShelleyActivated: boolean }) => {
+    try {
+      const generateWallet = (amount: number, reward: number): AdaWallet => {
```

Comment:
@daniloprates shouldn't we remove this part? I believe you added this only for testing purposes?

Reply:
@daniloprates I have removed it in this commit: https://github.com/input-output-hk/daedalus/pull/2111/commits/52b7cf69e5c26c262cb58e68d9554a737ae45576